Rest of text re: genetics of COVID-19 susceptibility (#1129)
greenelab/covid19-review: content/09.evolution.md

The region associated with this gene was not identified in the large-scale GWAS conducted by the COVID-19 Host Genetics Initiative (COVID-19 HGI) [@doi:10.1038/s41586-021-03767-x], which is described in more detail below.

A second source of genetic variability that was hypothesized to have an effect on COVID-19 outcomes were human leukocyte anitgens (HLA), or the major histocompatibility complex (MHC).
Both MHC classes I and II play a critical role in both the innate and adaptive immune system because they are a pivotal component of antigen presentation.
HLA classes I and II are also the most polymorphic loci in the human genome [@doi:10.1111/j.1399-0039.2012.01881.x].
Additionally, because HLA polymorphisms are associated with geographic ancestry, study location and participant background offers important context [@doi:10.1371/journal.pone.0097282].
Given the important role of the HLA complex in the immune response and the standing variation in the human population, HLA variation has been investigated for potential associations with COVID-19 outcomes.

Several approaches have been taken to evaluate a potential role of HLA in COVID-19.

> _In silico_ analysis suggested one particular HLA locus that could affect binding of SARS-CoV-2 peptides to MHC class II [@doi:10.1016/j.ijid.2020.07.016].<!-- HLA-DRB1*04:01,         HLA-DPA1*01:03/DPB1*02:01-->

Reviewer: What's with the greyed out bits?

Author: @RLordan I wrote the specific alleles in the comments in case they were useful in synthesizing the studies, I can remove them now!
```suggestion
_In silico_ analysis suggested one particular HLA locus that could affect binding of SARS-CoV-2 peptides to MHC class II [@doi:10.1016/j.ijid.2020.07.016].
```

Other studies evaluated outcomes using restrospective cohort analyses.
An analysis of 95 South Asian COVID-19 patients found that HLA genotype was not signficant in differentiating case severity when the necessary statistical corrections were applied [@doi:10.1002/jmv.27003].<!--risk:HLA-B*51, protect: HLA-B*35-->

> A study of a similar magnitude (n=147) in a European population did identify HLA alleles associated with severity [@doi:10.1111/tan.14284].<!--HLA-DRB1*04:01, also specifically DQA1*01:01-DQB1*05:01-DRB1*01:01-->

Reviewer: ```suggestion
Another study in a European population (n =147) did identify HLA alleles associated with severity [@doi:10.1111/tan.14284].<!--HLA-DRB1*04:01, also specifically DQA1*01:01-DQB1*05:01-DRB1*01:01-->
```

> In St. Louis, MO (USA), another study enrolled 234 COVID-19 cases, who were genotyped for HLA alleles and compared to a control population of 20,000 indiividuals from the National Marrow Donor Program [@doi:10.1111/tan.14387].

Reviewer: ```suggestion
In St. Louis, MO (USA), another study enrolled 234 COVID-19 cases, who were genotyped for HLA alleles and compared to a control population of 20,000 individuals from the National Marrow Donor Program [@doi:10.1111/tan.14387].
```

Finally, significant attention has been paid to the question of whether ABO blood type is associated with COVID-19 outcomes.<!--This paper is interesting but I'm not totally sure how to incorporate it 10.1371/journal.pone.0251535-->
ABO blood type has been found to modulate susceptibility to other pathogens [@doi:10.1001/jama.2020.16516].
While ABO blood type is a genetic trait, it is more easily evaluated than the genetic regions discussed above because of the simple relationship between genetic variants and phenotype.
The possibility for an association between blood type and COVID-19 infection was raised early in the pandemic in a preprint that reported associations in 2,173 patients in Wuhan and Shenzhen, China [@doi:10.1101/2020.03.11.20031096].
The protective effect of O and increased risk associated with A blood types that they reported was subsequently investigated by many studies that returned varied results (e.g., [@doi:10.1002/jmv.27444; @doi:10.3389/fcimb.2020.00404; @doi:10.1182/bloodadvances.2020002623; @doi:10.1001/jamanetworkopen.2021.7429]; see [@doi:10.1007/s00277-021-04489-w] for a literature review).

> Support for higher and lower risk, respectively, of SARS-CoV-2 infection with A and O blood types was supported by a meta-analysis [@doi:10.1016/j.meegid.2020.104485].

Reviewer: ```suggestion
Observations of higher and lower risk, respectively, of SARS-CoV-2 infection with A and O blood types was supported by a meta-analysis [@doi:10.1016/j.meegid.2020.104485].
```

Reviewer: Maybe pop the PloS paper here after the meta?

While the support for the association was independent of a mechanism, a possible relationship between ACE activity and blood type has been proposed [@doi:10.1177/2047487320922370] as has an effect on carbohydrate-carbohydrate interactions relevant to ACE2 binding [@doi:10.1016/j.mehy.2020.110155].
This is the only candidate gene described that has received additional support from GWAS, as is discussed below.

The COVID-19 literature related to candidate gene investigations demonstrates relatively low inter-study consistency in findings.
In particular, sample size is a major challenge in designing these studies.

> However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward.
> 
> As a result, in the age of next-generation sequencing, discovery-driven studies have emerged as an alternative approach.

Author: ```suggestion
However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward.
As a result, in the age of next-generation sequencing, discovery-driven studies have emerged as an alternative approach.
```

> However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward

Reviewer: ```suggestion
However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward.

```

> .

Reviewer: ```suggestion
```

##### Genome-Wide Association Studies

Genome-wide association studies (GWAS) offers a discovery-driven approach that provides a different perspective than candidate gene studies.

> Instead of selecting a gene or variant _a priori_, in GWAS, a large number SNPs (usually several million) are evaluated at once to identify those most likely to vary in correlation with a trait of interest.

Reviewer: ```suggestion
Instead of selecting a gene or variant _a priori_, in GWAS, a large number of SNPs (usually several million) are evaluated at once to identify those most likely to vary in correlation with a trait of interest.
```

Because of the large number of statistical tests, statistical power and multiple hypothesis testing are both very important considerations in executing GWAS, which have also struggled with issues related to replicability [@doi:10.1016/j.tig.2018.03.005].

> In cases such as COVID-19 where outcomes can differ among ancestry groups (likely for non-genetic reasons, as reviewed in [@individual-inequality]), it is especially important that GWAS samples be selected with attention paid to ancestry, as spurious associations can otherwise be identified with neutral markers indicative of ancestry itself [@doi:10.1038/nrg2813].

Reviewer: ```suggestion
In cases such as COVID-19 where outcomes can differ among ancestry groups (likely for non-genetic reasons, as reviewed in [@individual-inequality]), it is especially important that GWAS samples be selected with attention paid to ancestry, as incorrect or misleading associations can otherwise be identified with neutral markers indicative of ancestry itself [@doi:10.1038/nrg2813].
```

> In the past two years, many GWAS have been undertaken with the aim of identifying variants associated with COVID-19 outcomes.

Reviewer: ```suggestion
Over the past two years, many GWAS have been undertaken with the aim of identifying variants associated with COVID-19 outcomes.
```
